src/sdk/model/vm/guest-customization/guest-customization.ts

```diff
@@ -0,0 +1,104 @@
+import { GuestCustomizationJson } from './__json__';
+
+/**
+ * VM Guest Customization
+ */
+export class GuestCustomization {
+  protected _json: GuestCustomizationJson;
+
+  constructor(guestCustomization: GuestCustomization);
+  constructor(guestCustomizationJson: GuestCustomizationJson);
+  constructor(firstParam: GuestCustomization | GuestCustomizationJson) {
+    if (firstParam instanceof GuestCustomization) {
+      // Copy constructor
+      this._json = (firstParam as GuestCustomization).json;
+    } else {
+      // Json or empty constructor
+      this._json = (firstParam || {}) as GuestCustomizationJson;
+    }
+  }
+
+  get enabled(): boolean {
+    return this._json.enabled;
+  }
+
+  get changeSid(): boolean {
```

Comment:
In the case that this._json is empty, this getter won't return a boolean. You need to handle that case too i guess. Same will apply to other getters too...